### PR TITLE
base layout setup

### PR DESCRIPTION
**Pull request by bugudiramu**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+*,html,body{
+    border: none;
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+    font-family: 'IBM Plex Sans', sans-serif;
+}
+.layout{
+    background-color: red;
```
Comment (kkkumarkanike) [inline src/App.css:9]: is this red color used? if yes keep this, otherwise remove it.

File: src/components/PlatformLayout/PlatformLayout.tsx [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+          <div
+            style={{
+              height: 32,
+              margin: 16,
+              background: "rgba(255, 255, 255, 0.2)",
+              color: "#ffffff",
+              borderRadius: 8,
+              display: "flex",
+              justifyContent: "center",
+              alignItems: "center",
```
Comment (kkkumarkanike) [inline src/components/PlatformLayout/PlatformLayout.tsx:85]: avoid inline styling.

File: src/components/PlatformLayout/PlatformLayout.tsx [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+          }}
+        />
+      </Sider>
+      <Layout className="site-layout">
+        <Header style={{ padding: 0, background: colorBgContainer }}>
+          <div
+            style={{
+              marginLeft: 32,
+              paddingLeft: 8,
+            }}
```
Comment (kkkumarkanike) [inline src/components/PlatformLayout/PlatformLayout.tsx:110]: same here, avoid inline styling.

File: src/components/PlatformLayout/PlatformLayout.tsx [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+            </span>
+          </div>
+        </Header>
+        <Content style={{ margin: "0 16px" }}>
+          <Breadcrumb style={{ margin: "16px 0" }}></Breadcrumb>
+          <div
+            style={{
+              padding: 24,
+              minHeight: 360,
+              background: colorBgContainer,
```
Comment (kkkumarkanike) [inline src/components/PlatformLayout/PlatformLayout.tsx:128]: same here, avoid inline styling.

Comment (bugudiramu) [reply src/components/PlatformLayout/PlatformLayout.tsx:128]: resolved

**Review by kkkumarkanike — APPROVED** [review]

**Pull request merged**